JapaneseFilter/sections/general_elemhide.txt

```diff
@@ -130,6 +130,8 @@
 ##div[class^="google-2ad"]
 ##div[id^="active_overlay"]
 ##div[id^="post-"][class*="post-"] > div[id*="-"][class*="%e"]
+##a[rel="nofollow"][href*="/ban/pub/default.aspx?"]
```

Comment:
On some sites it will hide owl links.

Reply:
I guess we can add it and see how it goes.

Reply:
Nah. No one reports that sites.

Reply:
Ok.